src/pages/AuctionPage/AuctionPage.tsx

```diff
@@ -410,22 +413,42 @@ export const AuctionPageComponent: React.FC<AuctionPageProps> = ({ market }) =>
               <LineChart data={chartData} rangeSelector={rangeSelectorProps} />
             </Grid>
           )}
-          <Grid item sm={12} xs={12}>
+          {!isTablet && (
+            <Grid item sm={12} xs={12}>
+              <TradeHistory
+                columns={columnList}
+                rows={rows}
+                autoPageSize
+                title={t('bidHistory')}
+                disableSelectionOnClick
+                sortingOrder={['desc', 'asc', null]}
+              />
+            </Grid>
+          )}
+        </Grid>
+        {isTablet && (
+          <Grid item sm={12} xs={12} order={2} marginTop="1.5rem">
             <TradeHistory
-              columns={columnList}
+              columns={columnList.map((column) => ({
+                ...column,
+                headerAlign: 'left',
```

Comment:
im not sure if its coming from here but we should center the values in these rows i think
<img width="268" alt="Screen Shot 2021-10-01 at 13 34 18" src="https://user-images.githubusercontent.com/37285713/135613569-79d3c215-ea8b-4e55-ba21-a16fc50d76cb.png">

Reply:
in the design they are aligned left in all screens